tornado/testing.py

```diff
@@ -43,9 +43,9 @@
 import sys
 
 if PY3:
-    from io import StringIO
+    from io import StringIO  # noqa: F401
```

Comment:
Sounds good.

I noticed that some other tests have this same conditional import and might want to instead `from tornado.testing import StringIO` so that's why I initially left it.

Reply:
When we dropped python 2.6, I changed most of these conditional imports to an unconditional import of `io.BytesIO` or `io.StringIO`. I'm not sure why a few of the conditional versions are still around.